applications/nrf5340_audio/src/modules/led.h

```diff
@@ -42,8 +42,8 @@ enum led_color {
  * @note		If the given LED unit is an RGB LED, color must be
  *			provided as a single vararg. See led_color.
  *			For monochrome LEDs, the vararg will be ignored.
+ *			Using a LED unit assigned to another core will lead to undefined behavior.
```

Comment:
Not having the core specified anymore means that we cannot check if we are using a LED which is given to another core. I am following up with several stakeholders on this behavior. Performing a test, nothing happens when writing to a NET core LED from APP, but it would have been beneficial if we could get an error return from the gpio driver.

Reply:
I guess this is going to be continued outside of this PR?

Reply:
I have checked with stakeholders and changed the doc.